cpp-data-aggregation/aggregation_consumer.cpp

```diff
@@ -101,22 +123,30 @@ void run_aggregation_consumer(std::ofstream& timing_file,
 
 int main(int argc, char* argv[]) {
 
+    std::string context("Scaling tests");
```

Comment:
Add rank

Reply:
You dont mean the specific rank number right?

Reply:
I believe I did this correct